src/components/Features/Features.jsx

```diff
@@ -0,0 +1,183 @@
+import * as styles from './Features.module.scss';
+
+import React, { useState } from 'react';
+import {
+  collapsableList,
+  customerIcons,
+  featuresList,
+  frameworkIconsDotNet,
+  frameworkIconsJava,
+  frameworkIconsJavascript,
+  frameworkIconsOther,
+  frameworkIconsPhp,
+  frameworkIconsPython,
+  languageList,
+  navigationList,
+} from './dataSource.js';
```

Comment:
No need for extensions
```suggestion
} from './dataSource';
```